### PR TITLE
README instructions

### PR DESCRIPTION
**Pull request by panel-caleb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
-# iOS-interview
\ No newline at end of file
+# GIPHY iOS Code Challenge
+
+Please look at the challenge closely and make sure to cover all of the requirements.  The goal is to build a product that would reflect an app ready for production, but make sure to consider the tradeoffs necessary to complete the app in 90 minutes.
+
+## ⭐️ Technical Requirements
+- Swift is required
+- No SwiftUI, we love SwiftUI but we need to make sure you understand UIKit
```
Comment (panel-john) [inline README.md:7]: I know you said you're not comfortable with SwiftUI yourself but would you be comfortable if someone used it? even if you're not an expert with it

File: README.md [hunk 2]
```diff
@@ -1 +1,31 @@
...
+- Write the app as if it was going to be part of a larger scale application by focusing on architecture and best practices
+- Feel free to use any 3rd party libraries
+- Use any resources: Google, Stack Overflow, etc.  You'll need to explain your solution in depth, so be sure not to copy-paste
+
+## 🔍 What we are looking for
+- Project completeness
+- Code quality: style, organization, and design
+- Personal flare
```
Comment (panel-john) [inline README.md:15]: s/flare/flair

File: README.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+- Communication, collaboration, time management, and technical excellence
+
+Please submit your code to `caleb@getpanel.dev` after the interview.
```
Comment (panel-john) [inline README.md:18]: s/submit/email just to be explicit

**Review by panel-john — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (panel-caleb) [reply README.md:7]: You know, I thought about this, and I would be willing to let someone use SwiftUI if they could justify the benefit and tradeoff.  

I am happy to pair with someone using SwiftUI, I do know SwiftUI I am just not super fluent with it yet.

**Pull request merged**